py-orm/README.md

```diff
@@ -0,0 +1,78 @@
+Orm Sample App
+==============
+
+The orm sample app shows how to use the ``klotho::persist`` annotation with the `sqlAlchemy <https://www.sqlalchemy.org/>`_ Library.
```

Comment:
This looks like the python markdown-ish format (I forget what it's called). I don't think we want that, unless we change the file extension. But probably easier to just make it be markdown.

(DItto for the `::` code blocks below, etc).

Reply:
ah yeh i completely forgot about this, ty